New package: Anthropic v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Anthropic
- Repository: https://github.com/Cvikli/Anthropic.jl
- Created by: @Cvikli
- Version: v0.1.0
- Commit: 4c6c29b0b5bb870540695232bfde864f670c5f82
- Git reference: HEAD
- Description: Little extension for the https://github.com/svilupp/PromptingTools.jl to have streaming
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
unofficial Anthropic streaming support + some error handling
`````
<!-- END RELEASE NOTES -->

Cvikli commented:
compat issue resolved!

[noblock]

Cvikli commented:
Uploaded! Thank you for pointing out these guidelines!

[noblock]

Cvikli commented:
compat further issue resolved
[noblock]

goerz commented:
[noblock] You'll have to retrigger the registration the same way you created the original registration (without changing the version number!). That should then update this existing pull request

Cvikli commented:
Why are the tests are failing. I think I corrected. How can I rerun?

[noblock]

JuliaHubRegistrator committed:
New package: Anthropic v0.1.0

UUID: a00d14cb-3434-48ec-9b49-cfc2b9ce2a8f
Repo: https://github.com/Cvikli/Anthropic.jl.git
Tree: 6ef3b47d5562267da33c0aeb790a1b5ae8f674be

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

Cvikli commented:
> [noblock] You'll have to retrigger the registration the same way you created the original registration (without changing the version number!). That should then update this existing pull request

Thank you!
[noblock]
